client/app/scripts/superdesk/itemList/itemList.js

```diff
@@ -113,24 +113,10 @@ angular.module('superdesk.itemList', ['superdesk.search'])
 
         // Process related items only search
         if (options.related === true) {
-            var queryRelatedItem = [];
-            var queryWords = [];
-            queryWords = options.keyword.split(' ');
-            var length = queryWords.length;
-            queryRelatedItem.push('slugline' + ':("' + options.keyword + '")');
-
-            for (var i = 0; i < length; i++) {
-                if (options.keyword) {
-                    queryRelatedItem.push('slugline' + ':(' + queryWords[i] + ')');
-                }
-            }
-
-            if (queryRelatedItem.length) {
+            if (options.keyword) {
```

Comment:
We can combine this into a single `if` statement, saving a level of (unnecessary) indentation.